lib/chat/client/chat-ui.js

```diff
@@ -12,10 +12,9 @@ const { Logger, sc } = require('@reldens/utils');
 class ChatUi
 {
 
-    constructor(uiScene)
+    constructor(uiSceneManager)
     {
-        this.uiScene = uiScene;
-        this.gameManager = this.uiScene.gameManager;
```

Comment:
gameManager should remain, it should be passed to the constructor even if is not available on the uiScene, but since that should be replaced by our uiScene driver, it should recieve the game manager on it's own constructor as well

Reply:
same as before, let's discuss the implecations of giving a pluging access to all (GameManager has access to everything on the client side).

Reply:
keep in mind, GameManager is also available in the window context, go to your broser console and type: "reldens" :P 

everything in the browser is always available not matter how much you try to hide it or making it unavailable

in our case, though I don't use the global context, we need to make the GameManager available to every plugin

Reply:
Done.